assets/api-swagger/swagger.json

```diff
@@ -914,6 +914,228 @@
               }
             }
           }
+        }
+      },
+      "get": {
+        "tags": [
+          "Profiles"
+        ],
+        "summary": "Get the status of all merge jobs in a given account.",
```

Comment:
maybe just remove the word `all` since its paginated and that implies that you get everything in one go

Reply:
Sure.